Frends.AzureDataLake.UploadFiles/Frends.AzureDataLake.UploadFiles/Definitions/Input.cs

```diff
@@ -0,0 +1,22 @@
+namespace Frends.AzureDataLake.UploadFiles.Definitions;
+
+/// <summary>
+/// Optional parameters.
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Specify source files parameters
+    /// </summary>
+    public Source Source { get; set; }
+
+    /// <summary>
+    /// Specify destination Data Lake parameters
+    /// </summary>
+    public Destination Destination { get; set; }
+
+    /// <summary>
+    /// Specify options how we should handle data and errors
+    /// </summary>
+    public Options Options { get; set; }
```

Comment:
Let's move options to a separate tab (so task method is `(Input input, Options options, CancellationToken ...)`). This is mostly due to the fact that this is common practice in tasks.